wasm/integration-test/crypto/crypto.c

```diff
@@ -0,0 +1,114 @@
+#include "bindings_src/hermes.h"
+#include <string.h>
+
+// Exported Functions from `wasi:http/incoming-handler@0.2.0`
+void exports_wasi_http_incoming_handler_handle(exports_wasi_http_incoming_handler_own_incoming_request_t request, exports_wasi_http_incoming_handler_own_response_outparam_t response_out)
+{
+}
+
+// Exported Functions from `hermes:cardano/event-on-block`
+void exports_hermes_cardano_event_on_block_on_cardano_block(exports_hermes_cardano_event_on_block_cardano_blockchain_id_t blockchain, exports_hermes_cardano_event_on_block_cardano_block_t *block, exports_hermes_cardano_event_on_block_block_src_t source)
+{
+}
+
+// Exported Functions from `hermes:cardano/event-on-txn`
+void exports_hermes_cardano_event_on_txn_on_cardano_txn(exports_hermes_cardano_event_on_txn_cardano_blockchain_id_t blockchain, uint64_t slot, uint32_t txn_index, exports_hermes_cardano_event_on_txn_cardano_txn_t *txn)
+{
+}
+
+// Exported Functions from `hermes:cardano/event-on-rollback`
+void exports_hermes_cardano_event_on_rollback_on_cardano_rollback(exports_hermes_cardano_event_on_rollback_cardano_blockchain_id_t blockchain, uint64_t slot)
+{
+}
+
+// Exported Functions from `hermes:cron/event`
+bool exports_hermes_cron_event_on_cron(exports_hermes_cron_event_cron_tagged_t *event, bool last)
+{
+  return false;
+}
+
+// Exported Functions from `hermes:init/event`
+bool exports_hermes_init_event_init(void)
+{
+  return false;
+}
+
+// Exported Functions from `hermes:kv-store/event`
+void exports_hermes_kv_store_event_kv_update(hermes_string_t *key, exports_hermes_kv_store_event_kv_values_t *value)
+{
+}
+
+#define HERMES_STRING(x) \
+  (hermes_string_t) { (uint8_t *)x, strlen(x) }
+
+hermes_crypto_api_borrow_bip32_ed25519_t get_or_add_resource()
```

Comment:
```suggestion
hermes_crypto_api_borrow_bip32_ed25519_t create_resource()
```

Reply:
I am not renaming this for this PR,  if you want to enhance the test coverage it should be part of another PR ontop of this one.